### ISSUE TITLE
Suggestion to add a number or current date on youtube playlists 

### ISSUE THREAD

**Issue by helio-frota** [nodejs/admin#555]
Hi folks, 
I'm trying to watch the videos of N-API in series
https://www.youtube.com/watch?v=5QiaNFhVRoI&list=PLfMzBWSH11xb8_DRfdB3yiB4g7MPzGA_l&index=41

But it is unordered. 
Can you please add a number or current date on the title (I'm not sure how youtube works with playlists) but 
it is hard to follow what happen in the past.

Big thank you. And keep with the good work.

**Comment by mhdawson** [comment]
I typically add the date at the end of what Zoom gives us for the title. It would probably be easy to add at the start instead if it makes people easier to follow the sequence.  

While we discuss I'll try to remember to add at start for the next meetings I chair to see if there is any problem doing it. If not make sense to me to ask people chairing meetings across the org to do it.

**Comment by helio-frota** [comment]
Thank you. I believe whatever you folks do can help, because it is hard to follow, please look the order:

![2020-09-16_18-54](https://user-images.githubusercontent.com/6443576/93396458-19127900-f84e-11ea-9158-9f058457584c.png)

**Comment by mmarchini** [comment]
For the diag meetings I always add at the start, yyyy-mm-dd format.

**Comment by mhdawson** [comment]
I did DD-MM-YYYY for the N-API one today, we should agree on what we should recommend/use

**Comment by ljharb** [comment]
yyyy-mm-dd lexicographically sorts properly, and lacks ambiguity between US and Europe formats (obligatory https://xkcd.com/1179/)

**Comment by mhdawson** [comment]
Ok we should try to doc that somewhere `yyyy-mm-dd`

**Comment by helio-frota** [comment]
Thanks.
